config.json

```diff
@@ -1024,6 +1024,14 @@
       "uuid": "8d5aaf47-7503-4797-85ed-d35a8755cfd4"
     },
     {
+      "difficulty": 2,
+      "slug": "isbn-verifier",
+      "topics": [
+        "arrays"
+      ],
+      "uuid": "6b93edd9-0de4-4c80-6419-68b2f8232b91c3425fa"
+    },
```

Comment:
Since it's not core, we need to set a `unlocked_by`. @tleen your suggestion please ?

Reply:
`difference-of-squares`?

Reply:
Thanks!